agent/app.py

```
@@ -347,11 +430,12 @@ def start_server():
     """Start the FastAPI server."""
     config = Config.from_env()
     app = create_app()
-    
+
     uvicorn.run(
         app,
         host=config.ws_host,
         port=config.ws_port,
+        root_path=normalize_url_prefix(config.asgi_root_path),
```

Ruff (0.15.17)

[error] Undefined name `normalize_url_prefix`

(F821)